tests/test_handlers.py

```diff
@@ -243,6 +243,42 @@ def test_am1_bcc():
 
     assert vjp_fn(charges_adjoints) == None
 
+def test_am1_ccc():
+    
+    patterns = [
```

Comment:
I'd expand the test on a slightly more set of parameters and molecule.

Reply:
I could use the same molecule as in the rest of the tests.